### PR TITLE
feat: remove multiple bundle builds

### PR DESCRIPTION
**Pull request by HumbertoL**
This PR strips out the different entry points used to build cea-desktop. This change the build to have a single bundle that instead uses lazy loading to split chunks.

This other PR makes the necessary changes to cea-desktop to support a single bundle:
https://github.com/callemall/cea-desktop/pull/9820

To view all the changes in tea-react-scripts compared to the vanilla react-scripts, you can compare this branch to the official CRA repo
https://github.com/facebook/create-react-app/compare/main...callemall:create-react-app:single-bundle-scripts

### CODE REVIEW

File: packages/react-scripts/config/webpack.config.js [hunk 1]
```diff
@@ -97,68 +97,6 @@ const sassModuleRegex = /\.module\.(scss|sass)$/;
...
-const makeHtmlPluginEntryForBundle = (entryPoint, filename) =>
-  new HtmlWebpackPlugin({
-    inject: false,
-    chunks: [entryPoint],
-    template: entrypointTemplate,
-    filename: filename,
-    minify: false,
-    cache: false,
-  });
-// *** Call-Em-All Text-Em-All Web App
```
Comment (aywgit) [inline packages/react-scripts/config/webpack.config.js:161]: May want to keep this line 161 to indicate end of our additions in this block. Just `// *** Text-Em-All Web App`

File: packages/react-scripts/package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "tea-react-scripts",
-  "version": "1.0.16",
+  "version": "1.0.17",
```
Comment (aywgit) [inline packages/react-scripts/package.json:3]: Should update to `.18`

Comment (HumbertoL) [reply packages/react-scripts/package.json]: Thanks! Change pushed

**Review by aywgit — APPROVED** [review]
It's working!

**Comment by HumbertoL** [comment]
Shoot. I meant to merge the cea-desktop PR 🤦

I'll still address any feedback provided by Mike

**Comment by m2mathew** [comment]
It's all good. 🚢

**Pull request merged**